tests/umock_c_int/umock_c_int.c

```diff
@@ -1244,7 +1244,7 @@ TEST_FUNCTION(CopyOutArgumentBuffer_with_index_higher_than_count_of_args_trigger
     int injected_int = 0x42;
 
     // act
-    EXPECTED_CALL(test_dependency_1_out_arg(IGNORED_PTR_ARG))
+    EXPECTED_CALL(test_dependency_1_out_arg(IGNORED_ARG))
```

Comment:
>EXPECTED_CALL [](start = 4, length = 13)

wasn't this EXPECTED_CALL phased out in favor of the only STRICT_EXPECTED_CALL?
...

Reply:
And how is EXPECTED_CALL supposed to work with IGNORED_ARG versus STRICT_EXPECED_CALL with IGNORED_ARG? Is there a difference?

---
In reply to: [395825472](https://github.com/Azure/umock-c/pull/152#discussion_r395825472) [](ancestors = 395825472)

Reply:
It is phased out in our usage. EXPECTED_CALL is nothing else but syntactic sugar, equating STRICT_EXPECTED_CALL + IgnoreAllArguments.
So using IGNORED_ARG with EXPECTED_CALL does nothing additional.

---
In reply to: [395825969](https://github.com/Azure/umock-c/pull/152#discussion_r395825969) [](ancestors = 395825969,395825472)

Reply:
I won't remove now EXPECTED_CALL, will be a separate PR me thinks.

---
In reply to: [395836385](https://github.com/Azure/umock-c/pull/152#discussion_r395836385) [](ancestors = 395836385,395825969,395825472)